README.md

```diff
@@ -22,6 +22,14 @@ This program serves two purposes:
 2. Once we figure out all the details, it should be useful as an
    example of a systemd generator in rust.
 
+### Installation
+
+Copy the `zram-generator` binary into `/usr/lib/systemd/zram-generators/`,
+and `units/swap-create@.service` into `/usr/lib/systemd/system/`.
+
+The "{generator}" template in `units/swap-create@.service.d/binary-location.conf`
```

Comment:
Well, we're already defaulting to /usr for the generator, IMO it's easier to understand that you can copy it to /etc or /run or whatever if we say to copy *both* to /usr instead of distributing them over /usr and /etc